jsonxx_test.cc

```diff
@@ -560,7 +561,7 @@ int main(int argc, const char **argv) {
         o << "number" << 123;
         o << "string" << "hello world";
         o << "boolean" << false;
-        o << "null" << static_cast<void*>(0);
+        o << "null" << nullptr;
```

Comment:
changed from `void*` to `nullptr_t` because `void*` is now considered an invalid type (unless a serialization method is explicitly defined for it)